include/SFML/Graphics/Vertex.hpp

```diff
@@ -37,73 +37,24 @@ namespace sf
 ////////////////////////////////////////////////////////////
 /// \brief Define a point with color and texture coordinates
 ///
+/// By default, the vertex color is white and texture coordinates are (0, 0).
+///
 ////////////////////////////////////////////////////////////
-class Vertex
+struct Vertex
 {
-public:
-    ////////////////////////////////////////////////////////////
-    /// \brief Default constructor
-    ///
-    ////////////////////////////////////////////////////////////
-    constexpr Vertex();
-
-    ////////////////////////////////////////////////////////////
-    /// \brief Construct the vertex from its position
-    ///
-    /// The vertex color is white and texture coordinates are (0, 0).
-    ///
-    /// \param thePosition Vertex position
-    ///
-    ////////////////////////////////////////////////////////////
-    constexpr Vertex(const Vector2f& thePosition);
-
-    ////////////////////////////////////////////////////////////
-    /// \brief Construct the vertex from its position and color
-    ///
-    /// The texture coordinates are (0, 0).
-    ///
-    /// \param thePosition Vertex position
-    /// \param theColor    Vertex color
-    ///
-    ////////////////////////////////////////////////////////////
-    constexpr Vertex(const Vector2f& thePosition, const Color& theColor);
-
-    ////////////////////////////////////////////////////////////
-    /// \brief Construct the vertex from its position and texture coordinates
-    ///
-    /// The vertex color is white.
-    ///
-    /// \param thePosition  Vertex position
-    /// \param theTexCoords Vertex texture coordinates
-    ///
-    ////////////////////////////////////////////////////////////
-    constexpr Vertex(const Vector2f& thePosition, const Vector2f& theTexCoords);
-
-    ////////////////////////////////////////////////////////////
-    /// \brief Construct the vertex from its position, color and texture coordinates
-    ///
-    /// \param thePosition  Vertex position
-    /// \param theColor     Vertex color
-    /// \param theTexCoords Vertex texture coordinates
-    ///
-    ////////////////////////////////////////////////////////////
-    constexpr Vertex(const Vector2f& thePosition, const Color& theColor, const Vector2f& theTexCoords);
-
     ////////////////////////////////////////////////////////////
     // Member data
     ////////////////////////////////////////////////////////////
-    Vector2f position;            //!< 2D position of the vertex
+    Vector2f position{};          //!< 2D position of the vertex
```

Comment:
```suggestion
    Vector2f position;          //!< 2D position of the vertex
```
The default constructor already does what we want.

Reply:
I stand corrected! Removing `{}` here triggers `-Wmissing-field-initializers` in Clang and GCC. Much easier to keep these braces here since being explicit seems to make compilers happier.